ophyd/quadem.py

```diff
@@ -92,6 +92,36 @@ def __init__(self, *args, **kwargs):
         self.configuration_attrs = ['integration_time', 'averaging_time']
         self.read_attrs = ['current1.mean_value', 'current2.mean_value',
                            'current3.mean_value', 'current4.mean_value']
+        self._hints = None
+
+    @property
+    def hints(self):
+        """Provide hints to bluesky
+
+        The default value is ::
+
+           {'fields': list_of_channel_mean_value_names}
+
+        To override this, set another dictionary.
+
+        To restore the default value set ``None``
+
+        To suppress all hints set ``{}``
+        """
+        if self._hints is None:
+            return {'fields': [getattr(self, c).mean_value.name
+                               for c in ['current1', 'current2',
+                                         'current3', 'current4']]}
+        return self._hints
+
+    @hints.setter
+    def hints(self, val):
+        self._hints = val if val is None else dict(val)
+
+    def wtf(self):
```

Comment:
👍

Reply:
That should not have made it into a commit....

Reply:
The PR reviewer/merger sniggers, regretfully.